src/RollupAggregrid.js

```diff
@@ -272,6 +272,21 @@ Ext.define('Jarvus.aggregrid.RollupAggregrid', {
         me.ungroupSubRecords(staleRecords, false);
     },
 
+    onClick: function(ev, target) {
+        var me = this,
+            containerEl = me.el;
+
+        if ((target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) && target.up('.jarvus-aggregrid-subrow')) { // eslint-disable-line no-cond-assign
+            return me.onCellClick(
+                parseInt(target.up('.jarvus-aggregrid-subrow').getAttribute('data-subrow-id'), 10),
```

Comment:
@nbey you're calling `target.up('.jarvus-aggregrid-subrow')` twice, traversing the dom can be expensive, if this result was needed twice it should have been assigned to a variable